src/Android/Xamarin.Android/Samples/Layers/RasterRenderingRule/RasterRenderingRule.cs

```diff
@@ -0,0 +1,173 @@
+// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Views;
+using Android.Widget;
+using Esri.ArcGISRuntime.ArcGISServices;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Rasters;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+
+namespace ArcGISRuntimeXamarin.Samples.RasterRenderingRule
+{
+    [Activity]
+    public class RasterRenderingRule : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Create a global scope empty read-only list for the various rendering rules of the image service raster
```

Comment:
I don't like the wording of 'global scope' here. The object is scoped to the class and I don't think 'global' feels right in the context of .NET/C#.